docs/user/migrating_to_renku.rst

```diff
@@ -0,0 +1,142 @@
+fork or use as `cookiecutters <https://cookiecutter.readthedocs.io/en/latest/>`_.
+
+.. _migration_first_steps:
+
+Create an account on Renku & set up GitLab SSH key
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order to upload your project into Renku's web platform and the associated GitLab,
+you will need to create an account on `renkulab.io <https://renkulab.io>`_. There
+are several login options; pick the one most convenient for you.
+
+When you create a Renku account, a GitLab account gets created automatically for
+you. On this GitLab account (https://renkulab.io/gitlab), you will need to set
+up an SSH key to be able to clone and push code from your local machine. To do
+this:
+
+* visit http://renkulab.io/gitlab (while logged into `renkulab.io <https://renkulab.io>`_)
+* click the icon in the top righthand corner
+* click **Settings** from the dropdown
+* in the lefthand column, click **SSH Keys**
+* follow the instructions on this page
+
+Renku-ize your project with Renku CLI
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In order to Renku-ize your project, you'll need the Renku commandline interface
+(CLI). You can find the quick installation instructions
+`here <https://renku-python.readthedocs.io/en/latest/index.html>`_.
+
+.. warning:
+  If you don't already have python on your machine (but you do have git), it
+  might be easier to create a new project on `renkulab <http://renkulab.io>`_,
+  clone it locally, merge your existing project into this project, and push.
+
+If your project is not yet under version control (i.e. you haven't called
+``git init``), you can do the following (replace ``my_project`` with the name of
+your existing project)::
+
+  $ cd my_project
+  $ renku init
+
+If your project is *already* under version control, first make sure your
+``git status`` is clean, and inside the top level of your project run::
+
+  $ renku init --force
+
+As noted in the above section on :ref:`practical_renku_usage`, when you
+``renku init``'d, a number of directories, files, and dotfiles were added and
+committed in the top level of your directory.
+
+Now your project is ready to be sent to the web platform!
+
+Push your project to renkulab
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Pushing your project to Renku's GitLab instance boils down to adding the right
+remote and pushing the code. Perform the following, replacing ``<namespace>`` with
+the username that appears on the renkulab-generated GitLab instance (unless you
+are pushing to a group), and ``<project-name>`` with the name you are giving your project.
+
+If your project doesn't have a remote yet (check ``git remote``), you can use ``origin``::
+
+  $ git remote add origin git@renkulab.io/<namespace>/<project-name>.git
+  $ git push origin master
+
+If there is already a remote::
+
+  $ git remote add renku git@renkulab.io/<namespace>/<project-name>.git
+  $ git push renku master
+
+View your project on Renkulab
```

Comment:
please be consistent about capitalization of "Renkulab"

Reply:
which one do we want?

Reply:
not sure? since you are capitalizing Renku maybe use Renkulab?

Reply:
renkulab & Renku? renkulab & renku?

Reply:
going to go with lowercase for renkulab and capital for Renku because Renku is capitalized on renkulab.io, and lowercase for renkulab because otherwise we have to talk about whether to capitalize the L like JupyterLab does it.

Reply:
and additionally going to replace all random renkulabs with renkulab.io to be clearer